package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "loopback-example-ssl",
-  "version": "2.0.0",
+  "version": "3.0.0",
```

Comment:
Sorry, I wasn't very clear in my comment. I meant this ^ version... the version of this example repo (not of loopback or its modules). Typically we only change the major version of a repo when something big is changing that could cause backwards compatibility issues.